### ISSUE TITLE
Annulation de confirmation de présence

### ISSUE THREAD

**Issue by Tyraell** [addquebec/site#6]
Bonjour,

Dans le site actuel, il est impossible d'annuler une confirmation de présence. Les gens doivent appeler et nous créons un faux courriel indiquant "Annulation" avec le nom de la personne.

Devons nous ajouter une fonctionnalité pour ça?

Merci beaucoup et bonne journée,

**Comment by antoineleclair** [comment]
À discuter avec les autres probablement, mais de mon côté, je mettrais l'effort ailleurs pour l'instant. On pourrrait peut-être juste ajouter une phrase pas loin du formulaire qui dit "Pour annuler, écrivez à xxx@addquebec.ca"?

**Comment by Tyraell** [comment]
Oui, je suis d'accord avec toi. Le plus simple est de laisser ça de côté un peu pour le moment. Je ne crois pas que cela soit prioritaire.

De plus, ton idée d'ajouter une petite phrase comme tu as écris pourrait faire une bonne petite solution de contournement.

Mais avant de réaliser ceci, il faudrait s'assurer qu'il n'y a rien qui mentionne déjà cela sur le site. Je déjà regardé et je n'ai rien trouvé, mais j'avoue n'avoir pas chercher longtemps.

@DirectionADD Klo, qu'est-ce que tu en dit?

**Comment by DirectionADD** [comment]
Bonjour !! Oui je crois comme vous qu'une phrase serait préférable pour l'instant. Du genre:
Pour toutes demandes d'information ou annulation vous pouvez nous écrire directement à info@addquebec.ca, ou nous téléphoner au 581-300-6545.

**Comment by DirectionADD** [comment]
Je ne sais pas si cela est possible que les participants désirant annuler écrive directement sur resaddquebec@gmail.com ? Ou est-ce préférable de garder sur ce mail seulement les inscriptions...
Car sur info@addquebec.ca ce sera michelle qui recevra et qui devra par la suite en aviser le coach, et cela prend plus de temps...

**Comment by Tyraell** [comment]
Je crois que la phrase qui décrit la procédure d'annulation de la présence à un cours pourrait inciter la personne à écrire à la boite aux lettres resaddquebec@gmail.com. À partir de là, on peut créer une catégorie **Annulation** en rouge par exemple. De cette façon, les coachs pourraient voir facilement quand il y a une annulation sans passer par Michelle. On pourrait indiquer le cours et la personne en question dans le courriel.
